resources/src/IconActionSearch.tsx

```diff
@@ -0,0 +1,16 @@
+import {Search} from "@geist-ui/react-icons"
+import {Tooltip, useToasts} from "@geist-ui/react"
+
+const IconActionSearch = () => {
+    const [, setToast] = useToasts()
+
+    return <Tooltip text='Search'>
+        <Search
+            className='cursor-pointer'
+            size={28}
+            onClick={() => setToast({text: 'Use your browsers search functionality (Strg + F) to search'})}
```

Comment:
```suggestion
            onClick={() => setToast({text: 'Use your browser's search functionality (Strg + F) to search'})}
```
Also, what's `Strg`?

Reply:
<kbd>Strg</kbd> is the label for the [Control key](https://en.wikipedia.org/wiki/Control_key) in German keyboards. I would suggest using the more universally recognized <kbd>Ctrl</kbd> label.